packages/sitecore-jss-nextjs/src/components/EditingComponentPlaceholder.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
Technically, there's nothing Next.js-specific about this component so could be placed in `sitecore-jss-react` package. But think it's fine if we wanted to leave it here as well for now (it would be a pass-through anyway).